2.4/Dockerfile

```diff
@@ -37,7 +34,6 @@ RUN yum install -y epel-release && \
     yum install -y --setopt=tsflags=nodocs bind-utils gettext iproute v8314 mongodb24-mongodb mongodb24 && \
     yum clean all && \
     mkdir -p /var/lib/mongodb/data && chown -R mongodb.mongodb /var/lib/mongodb/ && \
```

Comment:
Will the image still work without chowning? I'm not sure about it. I think we should be able to run the image without privileges (so no nss switching necessary) and without volume and it should still work. I don't think it will, because it will not be able to write to /var/lib/mongodb.

Also, I'm taking back my comment about `-R`, it clearly is required.